src/platform/plugins/shared/dashboard/public/dashboard_api/unified_search_manager.ts

```diff
@@ -120,6 +121,19 @@ export function initializeUnifiedSearchManager(
   const controlGroupTimeslice$ = controlGroupApi$.pipe(
     switchMap((controlGroupApi) => (controlGroupApi ? controlGroupApi.timeslice$ : of(undefined)))
   );
+
+  // forward ESQL variables from the control group. TODO, this is overcomplicated by the fact that
```

Comment:
How about removing `TODO` comment. There are lots of places in this file where `controlGroupApi$` is consumed so this is not a unique problem for ESQL variables

Reply:
I wrote this `TODO`, and would prefer to either leave it, or to write an issue because I really think it would be great to simplify the interaction between the control group and the Dashboard.

Reply:
Lets go with an issue since there are multiple instances of this in the code base.

Reply:
Sure, I will write one up.